### PR TITLE
bluetooth-audio: Python 3 support / use separate dbus process

### PR DESCRIPTION
**Pull request by mglae**
Python 3 version of #3925 

There are still issues using dbus-python multi threaded, i.e. disabling bluetooth-audio addon terminates LibreELEC-settings  dbus bluetooth connection.

### CODE REVIEW

**Comment by lrusak** [comment]
I'm not very keen on this for a couple reasons:

1) why are you using subprocess.Popen to spawn the dbus service? Why not just do it in a thread?
2) reading from stdout seems like a hack to me. If you run the service in another thread you should be able to read from it or create a callback of some kind.

**Comment by mglae** [comment]
The assumption is dbus-python does not support installation from two threads of one process, in our case LibreELEC-settings and bluetooth-audio in kodi.

The process is created intentionally to avoid this limitation.

File: packages/addons/service/bluetooth-audio/source/default.py [hunk 1]
```diff
@@ -63,103 +59,38 @@ def __init__(self):
 
     xbmc.log('%s: starting add-on' % __addonid__, xbmc.LOGNOTICE)
```
Comment (gd71) [inline packages/addons/service/bluetooth-audio/source/default.py:60]: xbmc.LOGNOTICE not anymore supported, better change to xbmc.LOGINFO

File: packages/addons/service/bluetooth-audio/source/default.py [hunk 2]
```diff
@@ -63,103 +59,38 @@ def __init__(self):
...
-
-          xbmc.log('%s: bluetooth audio device disconnected' % __addonid__, xbmc.LOGNOTICE)
-          xbmc.log('%s: checking for other connected devices' % __addonid__, xbmc.LOGNOTICE)
+  def quit(self):
 
-          for path in self.devices:
-            if self.devices[path]['Connected'] and self.devices[path]['Class'] & (1 << 21):
-              xbmc.log('%s: found connected bluetooth audio device' % __addonid__, xbmc.LOGNOTICE)
-              return
+    xbmc.log('%s: stopping add-on' % __addonid__, xbmc.LOGNOTICE)
```
Comment (gd71) [inline packages/addons/service/bluetooth-audio/source/default.py:88]: xbmc.LOGNOTICE not anymore supported, better change to xbmc.LOGINFO

**Comment by mglae** [comment]
Thanks. Fixed and rebased.

**Comment by mglae** [comment]
Now depends on #4568.

**Review by CvH — APPROVED** [review]

**Pull request merged**